src/protocols/secure_channel/CASEServer.h

```diff
@@ -107,6 +107,9 @@ class CASEServer : public SessionEstablishmentDelegate,
      *
      */
     void PrepareForSessionEstablishment(const ScopedNodeId & previouslyEstablishedPeer = ScopedNodeId());
+
+    // If we are in the middle of handshake and receive a Sigma1 then respond with Busy status code.
+    void SendBusyStatusReport(Messaging::ExchangeContext * ec);
```

Comment:
Sending a Busy StatusReport(Busy being a General status code) seems like a general utility that all protocols could potentially use. From that standpoint, it might be useful to make it a public function usable by all protocols? It could be a static method with the ExchangeContext and wait time passed in?

Reply:
This is very a helper function for consumption of CASEServer only, so keeping it private.

If someone wants to send a busy status report they should use the `StatusReport::MakeBusyStatusReportMessage()` message builder and send using `ExchangeContext::SendMessage()`